### DISCUSSION TITLE
Population dynamics for a sessile organism?

### DISCUSSION THREAD

**Discussion by ZoologyDave** [RangeShifter/RangeshiftR-tutorials#14] [Q&A] [answered]
Heya, 

I'm planning a new project and trying to work out if RangeShiftR can do what I need (so apologies if this would be obvious through actually using the package): is it possible to model the population dynamics of sessile species e.g. trees, other plants etc. I couldn't work out if it was possible to have a simplified life history and dispersal process.

Thanks!

**Comment by JetteReeg** [comment, marked as answer]
Hi Dave, in general, it is possible to model population dynamics also for sessile species, e.g. by allowing dispersal only for seeds. However, RangeShiftR is not accounting for (biomass) growth per se as it only considers individuals in specific stages. So if you want to have e.g different fecundity for different plant sizes, you could define different stages representing different plant sizes.

**Reply by ZoologyDave** [reply]
Ah brilliant, sorry I missed that bit in the manual. Thanks a lot, Jette!
